Gems/ScriptCanvas/Code/Include/ScriptCanvas/AutoGen/ScriptCanvasGrammar_Source.jinja

```diff
@@ -26,12 +26,22 @@ SPDX-License-Identifier: Apache-2.0 OR MIT
 #include "{{ xml.attrib['Include'] }}"
 
 {%     for Class in xml.iter('Class') %}
+{% if Class.attrib['Base'] is defined %}
+{%      if Class.attrib['Base'].split(';')|length > 1 %}
+{%          set baseClass = Class.attrib['Base'].split(';')[0] %}
+{%          set baseClasses = ", ".join(Class.attrib['Base'].split(';')) %}
+{%      else %}
+{%          set baseClass = Class.attrib['Base'] %}
+{%          set baseClasses = Class.attrib['Base'] %}
+{%      endif %}
+{% else %}
+{%      set baseClass = "ScriptCanvas::Node" %}
+{%      set baseClasses = "ScriptCanvas::Node" %}
+{% endif %}
```

Comment:
this is duplicated, please apply the previous feedback and put it in the _macro.jinja otherwise it's a maintenance burden

Reply:
Note that at the moment jinja templates referenced in other jinja templates are not picked up by the compiler as a change, so changes in _macro.jinja wouldn't cause effected classes to be regenerated.

Reply:
That's ok because he still needs to change these templates to use the macros. That said, in this case the macro won't work, so he'll still need to do that, but I provided a smaller version of the change.